apps/src/applab/designElements/button.jsx

```diff
@@ -101,19 +104,17 @@ class ButtonProperties extends React.Component {
           handleChange={this.props.handleChange.bind(this, 'fontFamily')}
         />
         <PropertyRow
-          desc={'font size (px)'}
+          desc={applabMsg.designElementProperty_fontSizePx()}
           isNumber
           initialValue={parseInt(element.style.fontSize, 10)}
           handleChange={this.props.handleChange.bind(this, 'fontSize')}
         />
-        <EnumPropertyRow
-          desc={'text alignment'}
-          initialValue={element.style.textAlign || 'center'}
-          options={['left', 'right', 'center', 'justify']}
+        <TextAlignmentPropertyRow
+          initialValue={element.style.textAlign || TEXT_ALIGNMENT_CENTER}
```

Comment:
Could you make `TEXT_ALIGNMENT_CENTER` a default prop so you don't have to import `TEXT_ALIGNMENT_CENTER` and include this condition here, or are there other places where you want a default text alignment?

Reply:
Nevermind, I see you want left-alignment by default here: https://github.com/code-dot-org/code-dot-org/pull/52317/files#diff-6a122fbdbc1500797a40840273eb9246e910c68336b3f536ab2e227c561266e7R101.